HPXMLtoOpenStudio/resources/hpxml_defaults.rb

```diff
@@ -3696,6 +3697,21 @@ def self.apply_hvac_sizing(runner, hpxml_bldg, weather, output_format, design_lo
     HVACSizing.calculate(runner, weather, hpxml_bldg, hvac_systems, output_format: output_format, output_file_path: design_load_details_output_file_path)
   end
 
+  # Apply default ACH calculation for annual energy simulation when only leakiness description is provided, after Manual J sizing done
+  #
+  # @param hpxml_bldg [HPXML::Building] HPXML Building object representing an individual dwelling unit
+  # @return [void]
+  def self.apply_leakiness_description(hpxml_bldg)
+    measurement = Airflow.get_infiltration_measurement_of_interest(hpxml_bldg.air_infiltration_measurements)
+    return if measurement.unit_of_measure || measurement.effective_leakage_area
+
+    ach50 = Airflow.calc_ach50(hpxml_bldg)
+    measurement.house_pressure = 50
+    measurement.unit_of_measure = HPXML::UnitsACH
+    measurement.air_leakage = ach50
```

Comment:
You'll also want to set `measurement.air_leakage_isdefaulted = true` and ensure that the HPXML class uses it to assign the `dataSource` attribute for in.xml. It is important that a user see that the ACH50 value has been defaulted.

Reply:
Yes, I just did that today locally and made a few other changes trying to wrap up the development processes. Thanks for bringing this up!